src/django_twc_project/.github/workflows/test.yml.jinja

```diff
@@ -82,7 +82,7 @@ jobs:
 
       - name: Install dependencies
         run: |
-          python -m uv pip install --system -r requirements.txt -r requirements.dev.txt
+          python -m uv pip install --system --constraint requirements.txt --requirement pyproject.toml --extra dev
```

Comment:
TIL `constraint`

Reply:
I assume that this installs everything plus everything in `dev`? 

Asking because there used to be the opposite behavior where pip would only install `dev` (or so I thought).

Reply:
Yup, AFAIK it installs all the deps listed as primary and as well as the extras specified. Since they the majority aren't pinned in the list in `pyproject.toml`, the constraint on the versions listed in `requirements.txt` helps specify what to install.